sites/fast-creator/app/configs/fluent-ui/library.fluent-ui.design-system.mapping.ts

```diff
@@ -9,9 +9,427 @@ import {
     SwatchRGB,
 } from "@fluentui/web-components";
 
-export function setupFluentUIComponentDesignSystem(element: HTMLElement) {
+export function setupFluentUIComponentDesignSystem(
+    element: HTMLElement,
+    getAllFluentComponentTags: (containingSelector?: string) => string
+) {
     element.style.setProperty("background-color", `var(${fillColor.cssCustomProperty})`);
     element.style.setProperty("color", `var(${neutralForegroundRest.cssCustomProperty})`);
+    const fontFaceStyle = document.createElement("style");
+    fontFaceStyle.innerHTML = `
+        @font-face {
+            font-family: 'Leelawadee UI Web';
```

Comment:
@chrisdholt this is based off the Fabric React CSS, I am making the assumption that the web components will be matching them at some point and including all of the fonts here, let me know if that is not the case.

Reply:
We should go Segoe UI for now as that's still the base POR for Fluent UI I believe.

Reply:
Does Segoe UI account for languages other than Western European?

Reply:
Based on the Brand Central guidance, it should: https://microsoft.sharepoint.com/teams/brandcentral/Pages/The-Microsoft-brand-Core-elements-Type.aspx#topic_4

Reply:
These definitions seem a little out of date, since we know where the fonts are located based on the fluent react CSS, I'm going to leave them in and start an investigation so we can update them to be more accurate to brand guidance after MVP.